### DISCUSSION TITLE
Vulnerability - CVE-2019-1843

### DISCUSSION THREAD

**Discussion by inatnat** [typestack/class-validator#1885] [General]
Sorry, I know this topic have been discussed in different threads 

According to this link 
https://github.com/nestjs/nest/issues/10683

by setting forbidUnknownValues to true should have fixed the issue

However, our internal security scanning, vulnerability CVE-2019-1843 still mark exists, can anyone provide some idea how this can be fixed or even to provide fix in coming release?

**Comment by inatnat** [comment]
Dear all, may i get some help from anyone or give some us some idea to how to fix it?

**Comment by braaar** [comment]
This has been fixed. See https://github.com/typestack/class-validator/issues/1422
